### PR TITLE
Creating More Indicators for ART-REPORT

### PR DESCRIPTION
**Pull request by mozzy11**
indicators from  1 to 16.2

NB. most indicators still have dummy SQL files, we shall correct them later

cc @moshonk  and @ckemar

### CODE REVIEW

**Comment by moshonk** [comment]
Hi @mozzy11 Please push without force-push to preserve the comments and actions. It's rather challenging reviewing everything afresh anytime you push a commit. You can reserve the force-push action for the final commit.

File: api/src/main/resources/messages_fr.properties [hunk 1]
```diff
@@ -237,12 +237,29 @@ isanteplusreports.number_of_patients_with_malaria_hospitalized=Nombre de patient
...
+isanteplusreports.pnls.result.msm.label=MSM
+isanteplusreports.pnls.result.sexP.label=Professionnels du sexe		
+isanteplusreports.pnls.result.trans.label=transgenres		
+isanteplusreports.pnls.result.captv.label=prisonni�res
+isanteplusreports.pnls.result.drug.label=Utilisateurs de drogues
+isanteplusreports.pnls.result.newCtx.label=Patients VIH+ sous ARV nouvellement mis sous  CTX
+isanteplusreports.pnls.result.activeCtx.label=Patients actifs sous ARV sous CTX
+isanteplusreports.pnls.result.newTb.label=Nouveaux enrol�s sous ARV
+isanteplusreports.pnls.result.alreadyTb.label=Patients d�j� sous ARV 
 isanteplusreports.pnls.result.1-0.label=0-1 Yrs
```
Comment (moshonk) [inline api/src/main/resources/messages_fr.properties:263]: Please change the French label for _Yrs_ to _ans_ as in the excel template

**Comment by moshonk** [comment]
When you select multiple indicators, the output should be rendered in the same chronological order used to display them when selecting i.e. following the indicator numbers e.g. 1.1, 1.1.1, 1.2, 1.2.1 should follow one another in that order  @mozzy11

**Comment by mozzy11** [comment]
hello @moshonk   , sure that would be good , however ,
When Building the report ,
 unlike the Health Qual Report where all the report data is assigned to a single Dataset List  because all the indicators use the same one template hence are rendered based on the order of selection,

Here i  assign the report data to different Dataset Lists categorised by the output template , so reports are rendered based on the order oder of the Dataset List for a given template ,not necesarily in the order of selection especially for  indicators with different templates.

File: api/src/main/resources/messages_fr.properties (hunk 1 above)
Comment (mozzy11) [reply api/src/main/resources/messages_fr.properties]: done

File: api/src/main/resources/messages_es.properties [hunk 2]
```diff
@@ -54,3 +54,4 @@ isanteplusreports.pediatric11=Proportion of HIV-exposed infants who received ART
 isanteplusreports.pediatric12=Proportion of HIV-exposed infants between 4 weeks old and 12 months old who have received a PCR test
 isanteplusreports.pediatric13=Proportion of HIV-exposed infants who had a negative PCR test result during the selected period.
 isanteplusreports.pediatric14=Early Infant Diagnosis (EID).
+
```
[the remark this answers is not in the document]
Comment (mozzy11) [reply api/src/main/resources/messages_es.properties]: done

**Pull request merged**